packages/hooks-slas/README.md

```diff
@@ -0,0 +1,7 @@
+# SLAS Hooks
```

Comment:
Whatever name we choose, gotta make sure we remember to edit it here as well as in `package.json`.